app/src/app/[lng]/[inventory]/page.tsx

```diff
@@ -399,12 +378,33 @@ export default function Home({ params: { lng } }: { params: { lng: string } }) {
                                 N/A
                               </Text>
                             )}
-
-                            <InfoOutlineIcon
-                              w={3}
-                              h={3}
-                              color="brandScheme.100"
-                            />
+                            <Tooltip
+                              hasArrow
+                              label={
+                                <>
+                                  <Trans i18nKey="source-open-climate" t={t}>
+                                    {`Source: OpenClimate`}
+                                  </Trans>
+                                  <br />
+                                  <Trans
+                                    i18nKey="population-year"
+                                    values={{
+                                      year: inventory.city.populationYear,
+                                    }}
+                                    t={t}
+                                  >
+                                    {`Year: ${inventory.city.populationYear}`}
```

Comment:
This does not use the translation system's placeholders, but just JS string interpolation. Trans might fix this for you, but I think it might be nicer to have the placeholder here? Or alternatively just use the `t` function and passing the values in the second argument.